FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,38 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int64.h>
+#include <Arduino.h>
+std_msgs::Int64 fuel_level_percent_full;
+ros::Publisher pub_fuel_level("fuel_level_percent_full", &fuel_level_percent_full);
+ros::NodeHandle nh;
+
+const long LOOP_DELAY = 200;
+const int FUEL_GAUGE_READ_PIN = 61;
+const double VOLTAGE_ON_EMPTY = 2.45;
```

Comment:
It's good practice to be consistent in our naming. Like we end all pin declarations with _pin and we end all voltages with Voltage like you did in gauge_input_voltage
```suggestion
const double FUEL_EMPTY_VOLTAGE = 2.45;
```

Reply:
Doned